### PR TITLE
Add data functionalities to sanakoyeu_et_al_2018

### PR DESCRIPTION
**Pull request by pmeier**
Supersedes #162. @jbueltemeier merge if you are happy.

### CODE REVIEW

**Comment by pmeier** [comment]
Blocked by #178

File: pystiche_papers/sanakoyeu_et_al_2018/_data.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    return WikiArt(root, style, transform=transform, download=download)
+
+
+# TODO: replace this with torchvision.datasets.Places365 as soon as
+#  https://github.com/pytorch/vision/pull/2610 is part of a release
+def content_dataset(
+    root: str, transform: Optional[transforms.Transform] = None,
+) -> ImageFolderDataset:
+    if transform is None:
+        transform = image_transform()
```
Comment (jbueltemeier) [inline pystiche_papers/sanakoyeu_et_al_2018/_data.py:178]: I think I have mentioned this before, but the images of the Places365 are enlarged to twice the size again before this `image_transform`. See [here](https://github.com/CompVis/adaptive-style-transfer/blob/07a3b3fcb2eeed2bf9a22a9de59c0aea7de44181/prepare_dataset.py#L133).

Comment (pmeier) [reply pystiche_papers/sanakoyeu_et_al_2018/_data.py:178]: Only for these images or for all images (including style) in general?

Comment (jbueltemeier) [reply pystiche_papers/sanakoyeu_et_al_2018/_data.py:178]: This is only for the dataset Places365, but this is not mentioned in the paper.

**Review by jbueltemeier — APPROVED** [review]

**Pull request merged**